web/app/js/components/EdgesTable.jsx

```diff
@@ -117,10 +118,11 @@ EdgesTable.propTypes = {
   edges: PropTypes.arrayOf(processedEdgesPropType),
   namespace: PropTypes.string.isRequired,
   type: PropTypes.string.isRequired,
+  t: PropTypes.func.isRequired,
 };
 
 EdgesTable.defaultProps = {
   edges: [],
 };
 
-export default withStyles(styles)(EdgesTable);
+export default withTranslation(['Shared'])(withStyles(styles)(EdgesTable));
```

Comment:
I believe the standard here is `common` (at least that's what I've run into the most).